### DISCUSSION TITLE
Unable to find a way to push to AWS ECR,

### DISCUSSION THREAD

**Discussion by akil-rails** [quarkusio/quarkus#32918] [Q&A] [answered]
I am using the builder image: ubi-quarkus-mandrel-builder-image:22.3-java17 , and using jlib to build the container. 
1) On running without a push to the registry, the container build is successful, and the image is pushed to my local docker as per the [docs](https://quarkus.io/guides/container-image#jib).
2) When i try to push to the registry, I think I've to use [amazon-ecr-credentials-helper](https://github.com/awslabs/amazon-ecr-credential-helper), but this isn't installed on ubi-quarkus-mandrel-builder-image:22.3-java17.  i couldn't find a way to get this installed (tried microdnf, but there's no package for this).

Could anyone point me to some next steps i could try?

[a comment by an automated account is not shown]

**Comment by zakkak** [comment, marked as answer]
AFAIK the container image is created on the host, i.e. not in a container running `ubi-quarkus-mandrel-builder-image:22.3-java17`.

So if you are able to manually push the container image from your local docker to AWS ECR, I would expect you would be able to do the same through Quarkus. As a result, as a next step I would suggest trying to manually push the container image to AWS ECR to see if there are any issues with that.

**Reply by akil-rails** [reply]
Thanks, I didn't understand that the container images ends up in the target folder, i was further confused when i was trying with codebuild (aws) directly, where the host itself is a docker image. Works out of the box with the default configuration; (except that the privileged mode needs to be set to true for aws code build)
